tests/test_validation.py

```diff
@@ -0,0 +1,45 @@
+def test_put_empty(test_client):
+    response = test_client.put('/api/restaurants/0')
+    assert response.status_code == 400
+    assert b'{\n  "description": "Required data not available(No data)", \n  "name": "Bad request"\n}\n' \
+           == response.data
+
+
+def test_put_wrong_type(test_client):
+    response = test_client.put('/api/restaurants/0', json={"name": 5})
```

Comment:
test cases for the same method look quite similar, e.g. for `put` there are 2 things changed: request body and `description`. have you considered `pytest.mark.parametrize` as an option to run the same function with different arguments?

Reply:
fixed